ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/DirectionOrderMapper.java

```diff
@@ -48,13 +48,9 @@ public void from(CCDCase ccdCase, Claim.ClaimBuilder claimBuilder) {
         Optional.ofNullable(ccdCase.getPaperDetermination()).ifPresent(paperDetermination ->
             builder.paperDetermination(YesNoOption.valueOf(paperDetermination.name())));
 
-        Optional.ofNullable(ccdCase.getExpertReportPermissionPartyGivenToClaimant())
+        Optional.ofNullable(ccdCase.getGrantExpertReportPermission())
             .ifPresent(permission -> builder
-                .expertReportPermissionGivenToClaimant(YesNoOption.valueOf(permission.name())));
-
-        Optional.ofNullable(ccdCase.getExpertReportPermissionPartyGivenToDefendant())
-            .ifPresent(permission -> builder
-                .expertReportPermissionGivenToDefendant(YesNoOption.valueOf(permission.name())));
+                .grantExpertReportPermission(YesNoOption.valueOf(permission.name())));
```

Comment:
Is there a distinction between `NO` and `null` here?

Reply:
No. We are only interested in an explicit yes.